xcm/xcm-simulator/example/src/relay_chain.rs

```diff
@@ -140,7 +158,7 @@ pub type LocalOriginToLocation = SignedToAccountId32<Origin, AccountId, KusamaNe
 impl pallet_xcm::Config for Runtime {
 	type Event = Event;
 	type SendXcmOrigin = xcm_builder::EnsureXcmOrigin<Origin, LocalOriginToLocation>;
-	type XcmRouter = XcmRouter;
+	type XcmRouter = (XcmRouter, TestSendXcm);
```

Comment:
`TestSendXcm` might not be needed anymore, as `XcmRouter` always sends messages out.

Reply:
We need something in the test to track messages sent from the relay chain. I thought adding the `TestSendXcm` impl would work, but I see now that it does not. It short circuits on the first successful message sent.

This means I must change `XcmRouter` impl for the relay chain to store all messages sent.

Reply:
Instead of adding state to the macro, I think I will just change the `MockMsgQueue` pallet to store all received messages.